apps/swirl-docs/src/components/CommandMenu/CommandHit.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import { Command } from "cmdk";
+import { ReactNode } from "react";
+
+type CommandHitProps = {
+  title: string;
+  description?: string;
+  icon: ReactNode;
+  handleOnSelect: (value: any) => void;
+  handleOnFocus: (value: any) => void;
+};
+
+export function CommandHit({
+  title,
+  icon,
+  description,
+  handleOnSelect,
+  handleOnFocus,
+}: CommandHitProps) {
+  return (
+    <Command.Item onFocus={handleOnFocus} onSelect={handleOnSelect}>
+      <button
+        aria-label={title}
+        className={classNames(
+          "flex w-full h-full py-2",
+          "bg-surface-overlay-default",
+          "hover:bg-surface-hovered",
+          "focus:bg-surface-pressed",
+          "active:bg-surface-pressed",
+          "outline-none"
+        )}
+      >
+        <div className="inline-flex items-center max-w-5 pl-4 pr-3">{icon}</div>
```

Comment:
**issue:** Replace divs with spans. Block elements are not allowed inside buttons.